.storybook/webpack.config.js

```diff
@@ -34,8 +34,7 @@ module.exports = ({ config }) => {
             "@babel/plugin-proposal-nullish-coalescing-operator"
           ]
         }
-      },
-      require.resolve("react-docgen-typescript-loader")
```

Comment:
This package seems to be not maintained anymore (last change 3 years) and it was causing problems with typescript.
As far as I can see, it's for auto-generating documents from typescript comments, but I guess we don't need it at the moment. cc @mucahit